<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="datotekaPostajaUredjajLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Uredjaj :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelUredjaj">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>n/a</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Postaja :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPostaja">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>n/a</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1097</width>
         <height>533</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="siroviPodaciView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>250</height>
           </size>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="glavniTabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_ucitaj_podatke"/>
    <addaction name="action_otvori_kolektor"/>
    <addaction name="separator"/>
    <addaction name="action_spremi"/>
    <addaction name="action_ucitaj"/>
    <addaction name="separator"/>
    <addaction name="action_Napravi_pdf_report"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_ucitaj_podatke"/>
   <addaction name="action_otvori_kolektor"/>
   <addaction name="action_Napravi_pdf_report"/>
  </widget>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_ucitaj_podatke">
   <property name="text">
    <string>&amp;Ucitaj podatke</string>
   </property>
   <property name="toolTip">
    <string>Ucitaj sirove podatke iz datoteke</string>
   </property>
  </action>
  <action name="action_spremi">
   <property name="text">
    <string>&amp;Spremi kao...</string>
   </property>
   <property name="toolTip">
    <string>Spremi podatke i rezultate u file</string>
   </property>
  </action>
  <action name="action_ucitaj">
   <property name="text">
    <string>&amp;Ucitaj iz...</string>
   </property>
   <property name="toolTip">
    <string>Ucitaj podatke i rezultate iz filea</string>
   </property>
  </action>
  <action name="action_Napravi_pdf_report">
   <property name="text">
    <string>&amp;Napravi pdf report</string>
   </property>
  </action>
  <action name="action_otvori_kolektor">
   <property name="text">
    <string>&amp;Prikupljanje podataka</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
